### PR TITLE
chore: release v1.7.0, v1.6.1 and v1.5.2

### PR DESCRIPTION
**Pull request by bartsmykla**
### Summary

Release v1.7.0

### Full changelog

n/a

### Issues resolved

n/a

### Documentation

- ~~[ ] Link to the website [documentation PR](https://github.com/kumahq/kuma-website/pull/XXX)~~

### Testing

- [ ] Unit tests
- [ ] E2E tests
- [ ] Manual testing on Universal
- [ ] Manual testing on Kubernetes

### Backwards compatibility

- [ ] Update [`UPGRADE.md`](/UPGRADE.md) with any steps users will need to take when upgrading.
- ~~[ ] Add `backport-to-stable` label if the code follows our [backporting policy](/CONTRIBUTING.md#backporting)~~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,91 @@
...
+
+### New features:
+
+Gateway:
+* add `GatewayClass.Spec.ParametersRef` support [#4157](https://github.com/kumahq/kuma/pull/4157) @michaelbeaumont
+* cp annotations from gateway to svc [#4327](https://github.com/kumahq/kuma/pull/4327) @johnharris85
+* only reconcile Gateway when GatewayClass is Ready [#4162](https://github.com/kumahq/kuma/pull/4162) @michaelbeaumont
+
+Observability:
+* hijack application metrics [#4286](https://github.com/kumahq/kuma/pull/4286) @lukidzi
```
Comment (jakubdyszkiewicz) [inline CHANGELOG.md:13]: this is big, can we emphasize it a little bit? Something like
`hijack application metrics to enable scraping metrics from mTLSed applications without prometheus in the mesh`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,91 @@
 # CHANGELOG
 
+## [1.7.0]
+
+### New features:
+
+Gateway:
```
Comment (lahabana) [inline CHANGELOG.md:7]: I don't see arm here

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
...
+* bump github.com/gruntwork-io/terratest from 0.40.7 to 0.40.8 [#4260](https://github.com/kumahq/kuma/pull/4260) @dependabot
+* bump github.com/gruntwork-io/terratest from 0.40.8 to 0.40.10 [#4322](https://github.com/kumahq/kuma/pull/4322) @dependabot
+* bump github.com/lib/pq from 1.10.5 to 1.10.6 [#4299](https://github.com/kumahq/kuma/pull/4299) @dependabot
+* bump github.com/miekg/dns from 1.1.48 to 1.1.49 [#4291](https://github.com/kumahq/kuma/pull/4291) @dependabot
+* bump github.com/onsi/ginkgo/v2 from 2.1.3 to 2.1.4 [#4233](https://github.com/kumahq/kuma/pull/4233) @dependabot
+* bump github.com/prometheus/client_golang from 1.12.1 to 1.12.2 [#4290](https://github.com/kumahq/kuma/pull/4290) @dependabot
+* bump github.com/prometheus/common from 0.33.0 to 0.34.0 [#4235](https://github.com/kumahq/kuma/pull/4235) @dependabot
+* bump github.com/spf13/viper from 1.10.0 to 1.11.0 [#4177](https://github.com/kumahq/kuma/pull/4177) @dependabot
+* bump google.golang.org/grpc from 1.45.0 to 1.46.0 [#4213](https://github.com/kumahq/kuma/pull/4213) @dependabot
+* bump google.golang.org/grpc from 1.46.0 to 1.46.2 [#4289](https://github.com/kumahq/kuma/pull/4289) @dependabot
```
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md:86]: ack

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,91 @@
...
+* upgrade envoy to 1.22.0 [#4288](https://github.com/kumahq/kuma/pull/4288) @lobkovilya
+* upgrade kuma-cni to 0.0.10 [#4313](https://github.com/kumahq/kuma/pull/4313) @lobkovilya
+* upgrade tproxy iptables to v0.2.2 [#4328](https://github.com/kumahq/kuma/pull/4328) @bartsmykla
+* upgrade GUI to the latest version [#4316](https://github.com/kumahq/kuma/pull/4316) [#4338](https://github.com/kumahq/kuma/pull/4338) @jakubdyszkiewicz, @lahabana
+* upgrade kubectl to version with ARM support [#4180](https://github.com/kumahq/kuma/pull/4180) @lukidzi
+* upgrade protoc and regenerate files [#4169](https://github.com/kumahq/kuma/pull/4169) @lukidzi
+* bump github.com/golang-migrate/migrate/v4 from 4.15.1 to 4.15.2 [#4234](https://github.com/kumahq/kuma/pull/4234) @dependabot
+* bump github.com/gruntwork-io/terratest from 0.40.6 to 0.40.7 [#4178](https://github.com/kumahq/kuma/pull/4178) @dependabot
+* bump github.com/gruntwork-io/terratest from 0.40.7 to 0.40.8 [#4260](https://github.com/kumahq/kuma/pull/4260) @dependabot
+* bump github.com/gruntwork-io/terratest from 0.40.8 to 0.40.10 [#4322](https://github.com/kumahq/kuma/pull/4322) @dependabot
```
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md:78]: ack

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,91 @@
...
+* delete 'kubernetes.io/arch' node selector [#4335](https://github.com/kumahq/kuma/pull/4335) @lobkovilya
+
+CP:
+* don't always recompute mesh contexts [#4267](https://github.com/kumahq/kuma/pull/4267) @michaelbeaumont
+* don't run dataplane gc in global [#4184](https://github.com/kumahq/kuma/pull/4184) @lahabana
+* graceful components [#4277](https://github.com/kumahq/kuma/pull/4277) @jakubdyszkiewicz
+* memory store cannot delete a parent [#4194](https://github.com/kumahq/kuma/pull/4194) @jakubdyszkiewicz
+* protocol check should be case-insensitive [#4248](https://github.com/kumahq/kuma/pull/4248) @lukidzi
+* remove dns server from control plane [#4192](https://github.com/kumahq/kuma/pull/4192) @lahabana
+* fix: automatically detect dns lookup family for cp cluster [#4275](https://github.com/kumahq/kuma/pull/4275) @slonka
```
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md:51]: ack

File: CHANGELOG.md (hunk 1 above)
Comment (bartsmykla) [reply CHANGELOG.md:13]: ack

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,91 @@
...
+
+ZoneEgress:
+* resolve zone-ingress advertized address [#4219](https://github.com/kumahq/kuma/pull/4219) @lahabana
+* do not change ip to ZoneEgress address [#4193](https://github.com/kumahq/kuma/pull/4193) @lukidzi
+
+Kumactl:
+* remove flag '--experimental-meshgateway' [#4315](https://github.com/kumahq/kuma/pull/4315) @lobkovilya
+
+Other:
+* release for arm64 now publish correct arch image [#4276](https://github.com/kumahq/kuma/pull/4276) @lukidzi
```
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md]: ack

File: CHANGELOG.md (hunk 2 above)
Comment (bartsmykla) [reply CHANGELOG.md:20]: ack

File: CHANGELOG.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (bartsmykla) [reply CHANGELOG.md:20]: ack and done

File: CHANGELOG.md (hunk 1 above)
Comment (bartsmykla) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (bartsmykla) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,6 +1,140 @@
...
+* automate policy generation [#4197](https://github.com/kumahq/kuma/pull/4197) @lobkovilya
+
+## [1.6.1]
+> Released on 2020/06/10
+
+### Fixes:
+
+CP:
+* do not change ip to ZoneEgress address (backport #4193) [#4195](https://github.com/kumahq/kuma/pull/4195)
+* memory store cannot delete a parent (backport #4194) [#4196](https://github.com/kumahq/kuma/pull/4196)
```
Comment (lahabana) [inline CHANGELOG.md:130]: Missing: https://github.com/kumahq/kuma/pull/4232

Comment (bartsmykla) [reply CHANGELOG.md:131]: addressed

**Review by lahabana — APPROVED** [review]

**Pull request merged**